apps/api/src/database/seed/data-static.js

```diff
@@ -306,7 +306,7 @@ export const subSectors = [
 	{
 		subSector: {
 			name: 'training',
-			abbreviation: 'TRAIN01',
+			abbreviation: 'TRAI',
```

Comment:
the agreed prefix is TRAIN can you change it to that please

Reply:
Changed as requested and I have also had to allow for the 5 character TRAIN abbreviation as well as the usual 4 character abbreviations in the sql to handle the differing abbreviation lengths.